src/api/apiFetch.js

```diff
@@ -60,31 +61,56 @@ export const apiCall = async (
   }
 };
 
-export const apiGet = async (auth, route, params = {}, resFunc, options) =>
+export const apiGet = async (
+  auth: Auth,
+  route: string,
+  resFunc: (ApiResponse) => mixed,
+  params: Object = {},
+) =>
   apiCall(auth, `${route}?${encodeAsURI(params)}`, {
     method: 'get',
-  }, resFunc, options);
+  }, resFunc);
 
-export const apiPost = async (auth, route, params = {}, resFunc, options) =>
+export const apiPost = async (
+  auth: Auth,
+  route: string,
+  resFunc: (ApiResponse) => mixed,
```

Comment:
You can define a new type ResponseExtractor (ResponseExtractionFunc?) and reuse it.